build

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
```

Comment:
This seems harmless enough. I think that 2.7 has been out long enough that we can hopefully rely on it. Although RHEL just *barely* got their python upgraded to 2.7.

Perhaps it would work better if we just had it run "python2" instead of being specific about "python2.7"? My system has a symlink for python2. Does yours?